RevenueCatUI/UIKit/PaywallFooterViewController.swift

```diff
@@ -51,6 +51,25 @@ public final class PaywallFooterViewController: PaywallViewController {
     public required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    public override func loadView() {
+        super.loadView()
+
+        self.addChild(self.hostingController)
+        self.view.addSubview(self.hostingController.view)
+
+        NSLayoutConstraint.activate([
+            hostingController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            hostingController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            hostingController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+
+        self.hostingController.didMove(toParent: self)
+
+        hostingController.view.translatesAutoresizingMaskIntoConstraints = false
+        hostingController.view.backgroundColor = .clear
```

Comment:
This is probably not needed right?

Reply:
should have added a note, but it's needed so that you can actually see the cutout of the footer. otherwise the view from the hosting controller, which doesn't have the cutout and has a white background, will show its background behind the cutout